app/src/main/kotlin/io/imhungry/viewmodel/TagViewModel.kt

```diff
@@ -0,0 +1,40 @@
+package io.imhungry.viewmodel
+
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.LiveData
+import io.imhungry.db.ImhungryDatabase
+import io.imhungry.db.ImhungryRepository
+import io.imhungry.model.TagEntity
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.launch
+import kotlin.coroutines.CoroutineContext
+
+class TagViewModel(application: Application) : AndroidViewModel(application) {
```

Comment:
You don't need the application context in this view model. Remove the application parameter, and extend `ViewModel` instead of `AndroidViewModel`.